core/templates/pages/exploration-editor-page/modal-templates/lost-changes-modal.controller.spec.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for LostChangesModalController.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { LostChangeObjectFactory } from
+  'domain/exploration/LostChangeObjectFactory';
+import { TestBed } from '@angular/core/testing';
+import { UtilsService } from 'services/utils.service';
+// ^^^ This block is to be removed.
```

Comment:
I think these imports will stay when we migrate the corresponding controller to angular components. So we can get rid of these comments.

Reply:
Done.